.github/workflows/branch-pr-release.yaml

```diff
@@ -0,0 +1,32 @@
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+  release:
+    types:
+    - created
+jobs:
+  build:
+    strategy:
+      matrix:
+        os: [macos-latest, ubuntu-latest, windows-latest]
+    runs-on: ${{ matrix.os }}
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+    - name: Install Node.js
+      uses: actions/setup-node@v1
+      with:
+        node-version: 10.x
+    - run: npm install
+    - run: xvfb-run -a npm test
+      if: runner.os == 'Linux'
+    - run: npm test
+      if: runner.os != 'Linux'
+    - name: Publish
```

Comment:
https://github.com/marketplace/actions/github-action-for-vsce <- maybe look into this

Reply:
Should consider vetting and pinning a commit hash if we use that since it's third party

Reply:
I kept this consistent with the documentation.  I could rely on a 3rd party tool but doing it without it was so easy I just skipped it.